File: src/model/Trip.java
package model;

public class Trip {
    private final int id;
    private final String touristAttraction;
    private final String transportCompany;
    private final String departureTime;
    private final double price;
    private int availableSeats;

    public Trip(int id, String touristAttraction, String transportCompany, String departureTime, double price, int availableSeats) {
        this.id = id;
        this.touristAttraction = touristAttraction;
        this.transportCompany = transportCompany;
        this.departureTime = departureTime;
        this.price = price;
        this.availableSeats = availableSeats;
    }

    public int getId() { return id; }
    public String getTouristAttraction() { return touristAttraction; }
    public String getTransportCompany() { return transportCompany; }
    public String getDepartureTime() { return departureTime; }
    public double getPrice() { return price; }
    public int getAvailableSeats() { return availableSeats; }
    public void setAvailableSeats(int availableSeats) { this.availableSeats = availableSeats; }

    @Override
    public String toString() {
        return "Trip{id=" + id +
                ", attraction='" + touristAttraction + '\'' +
                ", company='" + transportCompany + '\'' +
                ", departure='" + departureTime + '\'' +
                ", price=" + price +
                ", seats=" + availableSeats + '}';
    }
}

File: TurismUI/TurismClient/Models/Trip.cs
namespace TurismClient.Models;

public sealed class Trip
{
    public int    Id                { get; }
    public string TouristAttraction { get; }
    public string TransportCompany  { get; }
    public string DepartureTime     { get; }
    public double Price             { get; }
    public int    AvailableSeats    { get; init; }

    public Trip(
        int    id,
        string touristAttraction,
        string transportCompany,
        string departureTime,
        double price,
        int    availableSeats)
    {
        Id                = id;
        TouristAttraction = touristAttraction;
        TransportCompany  = transportCompany;
        DepartureTime     = departureTime;
        Price             = price;
        AvailableSeats    = availableSeats;
    }

    public override string ToString() =>
        $"Trip {{ Id={Id}, Attraction='{TouristAttraction}', Company='{TransportCompany}', " +
        $"Departure='{DepartureTime}', Price={Price}, Seats={AvailableSeats} }}";
}
